test/unit/resources/aws_vpc_test.rb

```diff
@@ -16,14 +16,22 @@ def test_empty_params_ok
     AwsVpc.new
```

Comment:
Could you also update the examples in docs/resources/aws_vpc.md.erb?

Reply:
Done